test/unit/specs/table.spec.js

```diff
@@ -1758,6 +1758,52 @@ describe('Table', () => {
         });
       }, DELAY);
     });
+
+    it('sort correct change icon', done => {
+      function assertSortIconCount($el, msg, count = 1) {
+        const sortIconCount = $el.querySelectorAll('th.ascending, th.descending').length;
+        expect(sortIconCount).to.equal(count, msg);
+      }
+
+      const vm = createVue({
+        template: `
+          <el-table ref="table" :data="testData" >
+            <el-table-column prop="name" sortable />
+            <el-table-column prop="release" sortable />
+            <el-table-column prop="director" sortable />
+            <el-table-column prop="runtime" sortable />
+          </el-table>
+        `,
+
+        created() {
```

Comment:
Why do you set `testData` in the created hook? You can set `testData` in data function.
```js
data() {
  return { testData: getTestData() }
}
```

Reply:
You right, thanks.